ElementaryTasks/LogicalOperations.cs

```diff
@@ -6,107 +6,61 @@ namespace ElementaryTasks
 {
     class LogicalOperations
     {
-        public void FindMax()                    
+        public int FindMaxValue(int userInput1, int userInput2, int userInput3)                    
         {
-            Console.WriteLine("Enter the first value");
-            if (int.TryParse(Console.ReadLine(), out int maxValue))
+            int maxValue = userInput1;
+            if (userInput2 > maxValue)
             {
-                Console.WriteLine("Enter the second value");
+                maxValue = userInput2;
             }
 
-            if (int.TryParse(Console.ReadLine(), out int userInput2))
+            if (userInput3 > maxValue)
             {
-                Console.WriteLine("Enter the trird value");
+                maxValue = userInput3;
             }
-
-            if (int.TryParse(Console.ReadLine(), out int userInput3))
-            {
-                if (userInput2 > maxValue)
-                {
-                    maxValue = userInput2;
-                }
-
-                if (userInput3 > maxValue)
-                {
-                    maxValue = userInput3;
-                }
-            }
-
-            Console.WriteLine($"Value {maxValue} is the biggest");
+            return maxValue;
         }
 
-        public void EvenNmbers()                   
+        public int CalculateResult(int userInput1, int userInput2)                   
         {
-            Console.WriteLine("Enter the first value");
-            int.TryParse(Console.ReadLine(), out int userInput1);
-
-            Console.WriteLine("Enter the second value");
-            int.TryParse(Console.ReadLine(), out int userInput2);
-
             if ((userInput1 % 2) == 0)
             {
-                Console.WriteLine($"Result: {userInput1 * userInput2}");
+                int result = userInput1 * userInput2;
+                return result;
             }
             else
             {
-                Console.WriteLine($"Result: {userInput1 + userInput2}");
+                int result = userInput1 + userInput2;
+                return result;
             }
         }
-        public void PointCoordinates()            
+        public string FindCoordinatesOfPoints(int x, int y)            
         {
-            Console.WriteLine("Enter the x value");
-            if (!int.TryParse(Console.ReadLine(), out int x))
-            {
-                Console.WriteLine("Error! You should enter numerical value");
-                return;
-            }
-
-            Console.WriteLine("Enter the y value");
-            if (!int.TryParse(Console.ReadLine(), out int y))
-            {
-                Console.WriteLine("Error! You should enter numerical value");
-                return;
-            }
-
+            string plane;
             if (x > 0 && y > 0)
             {
-                Console.WriteLine($"Coordinates of the point ({x},{y}) are in I plane");
+                plane = "I";
+                return plane;
```

Comment:
You should't write return in each if block.
The only one return required at the end of method body.

Reply:
Please check all methods

Reply:
Done